app/assets/javascripts/backbone/views/expander.coffee

```diff
@@ -11,26 +11,38 @@ class MS.Views.Expander extends Backbone.View
 
   expand: (e) ->
     e.preventDefault()
-    link = @$(e.currentTarget)
-    targetName = link.data('expands')
+    $link = @$(e.currentTarget)
```

Comment:
Can you please update the comment at the top of this file to describe:

1. expandAll/hideAll (these never got added to the comment)
2. The nested expander scenario (that you have expandables within an expandable and you want to be able to expand all the inner expandables with one click)

I think that would make this easier to understand later.